public/data/vaccinations/country_data/India.csv

```diff
@@ -93,6 +93,7 @@ India,2021-04-16,"Covaxin, Oxford/AstraZeneca",http://mohfw.gov.in/pdf/Cumulativ
 India,2021-04-17,"Covaxin, Oxford/AstraZeneca",http://mohfw.gov.in/pdf/CumulativeCOVIDVaccinationCoverageReport17thApril2021.pdf,122622590,106431076,16191514
 India,2021-04-18,"Covaxin, Oxford/AstraZeneca",http://mohfw.gov.in/pdf/CumulativeCOVIDVaccinationCoverageReport18thApril2021.pdf,123852566,107371770,16480796
 India,2021-04-19,"Covaxin, Oxford/AstraZeneca",http://mohfw.gov.in/pdf/CumulativeCOVIDVaccinationCoverageReport19thApril2021.pdf,127129113,109659181,17469932
+India,2021-04-20,"Covaxin, Oxford/AstraZeneca",http://mohfw.gov.in/pdf/CumulativeCOVIDVaccinationCoverageReport20thApril2021.pdf,130119310,111645892,18473418
```

Comment:
Thanks for your help.

For now, we are not considering this entry as it would not be consistent  with subsequent values (the number of total vaccinations and people fully vaccinated were lower on the 2021-0421)